### ISSUE TITLE
Add Scrollback for joining users.

### ISSUE THREAD

**Issue by zorchenhimer** [zorchenhimer/MovieNight#142]
Add a scrollback with the last N messages in chat to send to joining users.  The number of messages should be configurable.

**Comment by zorglube** [comment]
Yep, seems to be an good idea. 
Did you already though of a solution to keep the last `N` messages ?

**Comment by CptPie** [comment]
if i understand correctly a chat message is defined here: https://github.com/zorchenhimer/MovieNight/blob/9ba472bb5b26b8849b3fe44580f601a01cd8526a/common/chatdata.go#L86
Therefore it should be rather easy to implement a rolling stack with a fixed length to enable scrollback.

**Comment by zorglube** [comment]
I bet the 'bigest' problem isn't the stack but handling the new chat user and send just to him the stack content.

**Comment by zorchenhimer** [comment]
Nah, that should be pretty straight forward.  A bunch of (hidden) messages are already sent to the client during the join process.  Sending the scrollback would just happen after all that.  I suppose it could be a new hidden message type so it could be formatted differently or something (eg, greyed out), but that's not exactly necessary.
